layouts/post/single.html

```diff
@@ -17,5 +17,6 @@ <h1 class="post-title">{{ .Title }}</h1>
 {{ end }}
 
 {{ .Content }}
-  
-{{ end }}
\ No newline at end of file
+{{ template "_internal/disqus.html" . }}
+
+{{ end }}
```

Comment:
let the user configure whether to enable disqus or not. something like

```
{{ if eq site.Params.enableDisqus true }}
  {{ template "_internal/disqus.html" . }}
{{ end }}
```

then add a `enableDisqus` param to the `config.toml` and `config.yaml` files in the exampleSite

Reply:
@zwbetz-gh
The Disqus is enabled when `disqusShortname` param is declared and `privacy.disqus.disable` is not `false`.
These options can be found in the following links.

* https://gohugo.io/content-management/comments/
* https://gohugo.io/about/hugo-and-gdpr/#all-privacy-settings
* https://github.com/gohugoio/hugo/blob/master/tpl/tplimpl/embedded/templates/disqus.htm

Because there are already the options supported by Hugo, it would be complicated to create another parameter.

I added these options to the config.toml and config.yaml files in the exampleSite. ( https://github.com/benelog/papercss-hugo-theme/commit/995c90dc37e057434d821f2f45d8b376c646137a )

Reply:
Thanks for the info. Yeah since there is an existing option let's use that. I also like how you set `disable` to `true` by default. Merging this.